Gruntfile.js

```diff
@@ -119,24 +118,46 @@ module.exports = function(grunt) {
     },
 
     clean: {
-      build: [
-        'build/*.js',
-        'build/*.map',
-        'dist/*.js',
-        'dist/*.map'
-      ],
+      build: ['build/**/*', '!build/.gitkeep'],
+      dist: ['dist/*', '!dist/.gitkeep'],
     },
 
     copy: {
-      main: {
-        cwd: 'build/',
+      lib: {
+        cwd: 'build/lib/',
         src: ['*.js', '*.map'],
         dest: 'dist/',
         filter: 'isFile',
         expand: true,
+      },
+      'example-packaged': {
+        cwd: 'example/packaged/',
+        src: '*',
+        dest: 'build/app/',
+        expand: true,
+      },
+      'example-shared': {
+        cwd: 'example/shared/',
+        src: '**/*',
+        dest: 'build/app/',
+        expand: true,
+      },
+      'lib-to-package': {
+        cwd: 'build/lib/',
+        src: '*',
+        dest: 'build/app/',
+        expand: true,
       }
     },
 
+    zip: {
+      app: {
+        cwd: 'build/app/',
+        src: 'build/app/**',
+        dest: 'build/app.zip',
```

Comment:
the packaged example readme mentions ``build/application.zip``. I don't mind which one you change.